components/asset/introduction.rst

```diff
@@ -143,13 +153,15 @@ every day::
         }
     }
 
+.. _component-assets-path-package:
+
 Grouped Assets
 ~~~~~~~~~~~~~~
 
-It's common for applications to store their assets in a common path. If that's
-your case, replace the default :class:`Symfony\\Component\\Asset\\Package` class
-by :class:`Symfony\\Component\\Asset\\PathPackage` to avoid repeating the same
-path time and again::
+Often, many assets live under a common path (e.g. ``/static/images``). If
+that's your case, replace the default :class:`Symfony\\Component\\Asset\\Package`
+class with :class:`Symfony\\Component\\Asset\\PathPackage` to avoid repeating
+that path time and again::
```

Comment:
Actually, `to avoid repeating that path over and over again` is much clearer in native English. Thanks for pointing this out :)